react-js/react-components-and-hierarchies.md

```diff
@@ -34,36 +35,31 @@
 - [How to pass data to props.children](https://frontarm.com/james-k-nelson/passing-data-props-children/)
 - [React Strap](https://reactstrap.github.io/) 
 
-
-<br/>
-
 ### Lesson
 
-#### What is the difference between Components and built-in JSX tags like <button>
+#### What is the difference between Components and built-in JSX tags like button ?
 
-Difference between Components and built-in JSX tags are :
-- Components are created by the user while JSX tags are pre defined
-- Components are usually self closing in nature (eg: ```<App/>, <Header/>```) while tags are in pairs(```<button>this is a tag</button>, <p>this is a paragraph tag </p>```)
+The difference between Components and built-in JSX tags is that components are created by the user while JSX tags are pre-defined
 
 Consider the following example
 ```
-function App() {
-  return <h1>Hello World  </h1>;
+class App extends React.Component {
+   render() {
+     return <h1>Hello World  </h1>;
+  }
 }
 
 ReactDOM.render(
   <App />,
   document.getElementById('root')
 );
 ```
-Here we have functional *component named App*, that returns 'Hello World ' inside a *h1 tag*. 
-<br/><br/>
+In this example, App is a component while h1 is a built-in JSX tags
 
 
+### How to pass data Between React Components
```

Comment:
Please fix the casing of this and all headings on this page. Headings typically use [Title Case](https://www.bkacontent.com/how-to-correctly-use-apa-style-title-case/)